src/main/java/com/robertx22/age_of_exile/database/data/unique_items/UniqueGear.java

```diff
@@ -70,6 +74,11 @@ public UniqueGear fromJson(JsonObject json) {
                     .getAsBoolean();
         }
 
+        if (json.has("flavor_text")) {
+            uniq.flavorText = json.get("flavor_text")
+                    .getAsString();
```

Comment:
add a flavor text mechanic to Unique, but I don't know if it can work by using the JSON reg.

Reply:
It should work but it needs a way to check if the flavor text is missing (if the  SlashRef.MODID + ".unique_gear." + this.GUID() + ".flavor_text"; ) translatable is there or not in the lang file. Otherwise i'd have to add flavor to every unique